<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewItem</class>
 <widget class="QDialog" name="DialogNewItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonNext">
            <property name="text">
             <string>Suivant</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButtonCancel">
            <property name="text">
             <string>Annuler</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QGroupBox" name="groupOptions">
            <property name="title">
             <string>Options</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_6">
               <property name="verticalSpacing">
                <number>20</number>
               </property>
               <item row="1" column="0">
                <widget class="QRadioButton" name="radioMap">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Nouvelle Carte
Une carte est capable d'afficher une image support ainsi que des marqueurs.</string>
                 </property>
                 <property name="icon">
                  <iconset resource="Images.qrc">
                   <normaloff>:/new/Icons/Plan.png</normaloff>:/new/Icons/Plan.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QRadioButton" name="radioSce">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Nouveau Scénario
Un scénario est composés de styles et de cartes.</string>
                 </property>
                 <property name="icon">
                  <iconset resource="Images.qrc">
                   <normaloff>:/new/Icons/sce.png</normaloff>:/new/Icons/sce.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QRadioButton" name="radioSty">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Nouveau Style
Les styles permettent d'ajouter des marqueurs avec une icône personnalisée.</string>
                 </property>
                 <property name="icon">
                  <iconset resource="Images.qrc">
                   <normaloff>:/new/Icons/ButtonDeBase</normaloff>:/new/Icons/ButtonDeBase</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="2" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButtonPrevious">
          <property name="text">
           <string>Précédent</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QGroupBox" name="groupImage">
          <property name="title">
           <string>Image</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_14">
             <item row="0" column="0">
              <widget class="QLineEdit" name="lineImage"/>
             </item>
             <item row="0" column="1">
              <widget class="QToolButton" name="toolButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QGraphicsView" name="graphicsView"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QGroupBox" name="groupName">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Nom</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLineEdit" name="lineName"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButtonCancel2">
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButtonOk">
          <property name="text">
           <string>Terminé</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewItem</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel2</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewItem</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewItem</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
